packages/deployment/app/config.yml

```diff
@@ -29,10 +29,18 @@ default:
   shared_data_bucket: 'cumulus-data-shared'
 
   buckets:
-    internal: change-me
-    private: change-me
-    protected: change-me
-    public: change-me
+    internal: 
```

Comment:
Level of confidence in not having any assumptions of particular bucket names, given that the names and types are all equal?

Reply:
@bilts I have scoured the code for the word buckets. The only thing I have left is the distribution API uses the protected bucket - could this be the system bucket?

Reply:
I would think that the distribution API assumes that all data is in the same protected bucket, which is no longer true.  Are you able to distribute data that was ingested into a bucket not called "protected"?